packages/wdio-globals/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import '../types.d.ts'
+/// <reference path="../types.d.ts" />
```

Comment:
@christian-bromann  I see that in the end this part did not come up in the patch for the latest version, why? This is what makes patching work and make types work

Reply:
> This is what makes patching work and make types work

I don't think so. If you create a new WebdriverIO project through `npm init wdio .` you will see that types are propagated correctly if the setup follows our [configuration recommendation](https://webdriver.io/docs/typescript). Please provide a reproducible example if you don't get this.